CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ReplicatorConfigurationRequestHandler.swift

```diff
@@ -228,26 +228,31 @@ public class ReplicatorConfigurationRequestHandler {
             var replicatorConfiguration: ReplicatorConfiguration = args.get(name: "configuration")!
             let authenticator: Authenticator = args.get(name: "authenticator")!
             replicatorConfiguration.authenticator = authenticator
+            return replicatorConfiguration
```

Comment:
is it required to to return this for each method? I do not see any description for this PR. When you create replicator configuration , you can use that object, you do not require to return for each method



Explaining little is better, so that I can understand why changes required

Reply:
Done, it is the same reason for python test change that I request along with this PR